taskchampion/taskchampion/src/replica.rs

```diff
@@ -113,17 +116,40 @@ impl Replica {
     /// although previously-returned dependency maps are not updated.
     pub fn dependency_map(&mut self, force: bool) -> anyhow::Result<Rc<DependencyMap>> {
         if force || self.depmap.is_none() {
+            // note: we can't use self.get_task here, as that depends on a
+            // DependencyMap
+
             let mut dm = DependencyMap::new();
+            // temporary cache tracking whether tasks are considered Pending or not.
+            let mut is_pending = HashMap::new();
```

Comment:
I would find this more clear if the fact that it was a cache was part of the name and if the types were explicit:

```rs
let mut is_pending_cache = HashMap::<Uuid, bool>::new();
```

The comment would then be harmless but perhaps superfluous.

Reply:
Done :)

Reply:
Can we get the explicit types too?

Reply:
Oh, I missed that.